### PR TITLE
Added the supported API classes

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Fixes: #91

### CODE REVIEW

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc [hunk 1]
```diff
@@ -226,6 +226,65 @@ Implementations MAY support:
...
+
+### Semantic Conventions
+
+[NOTE]
+====
+These packages are not stable, which might be subject to changes.
+
+- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-semconv/1.28.0-alpha/io/opentelemetry/semconv/trace/attributes/package-summary.html[io.opentelemetry.semconv.trace.attributes]
+- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-semconv/1.28.0-alpha/io/opentelemetry/semconv/resource/attributes/package-summary.html[io.opentelemetry.semconv.resource.attributes]
+====
```
Comment (Azquelt) [inline tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc:286]:
```suggestion
[NOTE]
====
These packages are not stable and may be subject to breaking changes in future releases.
====

- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-semconv/1.28.0-alpha/io/opentelemetry/semconv/trace/attributes/package-summary.html[io.opentelemetry.semconv.trace.attributes]
- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-semconv/1.28.0-alpha/io/opentelemetry/semconv/resource/attributes/package-summary.html[io.opentelemetry.semconv.resource.attributes]
```

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc [hunk 2]
```diff
@@ -226,6 +226,65 @@ Implementations MAY support:
...
+- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/1.28.0/io/opentelemetry/sdk/autoconfigure/spi/traces/package-summary.html[io.opentelemetry.sdk.autoconfigure.spi.traces]
+
+The above packages have dependencies on the following packages:
+
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/package-summary.html[io.opentelemetry.sdk.trace]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/data/package-summary.html[io.opentelemetry.sdk.trace.data]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/export/package-summary.html[io.opentelemetry.sdk.trace.export]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/samplers/package-summary.html[io.opentelemetry.sdk.trace.samplers]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-common/1.28.0/io/opentelemetry/sdk/common/package-summary.html[io.opentelemetry.sdk.common]
```
Comment (Azquelt) [inline tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc:272]:
```suggestion
The above packages have dependencies on the following packages which must be supported to the extent that they are required by the Autoconfigure SPI classes:

  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/package-summary.html[io.opentelemetry.sdk.trace]
  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/data/package-summary.html[io.opentelemetry.sdk.trace.data]
  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/export/package-summary.html[io.opentelemetry.sdk.trace.export]
  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/1.28.0/io/opentelemetry/sdk/trace/samplers/package-summary.html[io.opentelemetry.sdk.trace.samplers]
  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-common/1.28.0/io/opentelemetry/sdk/common/package-summary.html[io.opentelemetry.sdk.common]
```

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc [hunk 3]
```diff
@@ -226,6 +226,65 @@ Implementations MAY support:
...
+
+### `WithSpan` annotation
+
+- https://www.javadoc.io/doc/io.opentelemetry.instrumentation/opentelemetry-instrumentation-annotations/latest/io/opentelemetry/instrumentation/annotations/package-summary.html[io.opentelemetry.instrumentation.annotations]
```
Comment (Azquelt) [inline tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc:276]:
```suggestion
- https://www.javadoc.io/doc/io.opentelemetry.instrumentation/opentelemetry-instrumentation-annotations/latest/io/opentelemetry/instrumentation/annotations/package-summary.html[io.opentelemetry.instrumentation.annotations] (`WithSpan` and `SpanAttribute` only)
```

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc [hunk 4]
```diff
@@ -226,6 +226,65 @@ Implementations MAY support:
 To obtain the `Tracer` with the OpenTelemetry API, the consumer must use the exact same instrumentation name and version used by the implementation.
 Failure to do so, may result in a different `Tracer` and incorrect handling of the OpenTelemetry data.
 
+
+=== OpenTelemetry API Classes supported by this specification
```
Comment (Azquelt) [inline tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc:230]:
```suggestion
=== Supported OpenTelemetry API Classes
```

**Review by Azquelt — COMMENTED** [review]
Looks good, I've suggested some rewording and some small limitations where we don't require support for every class within a package.

**Comment by Emily-Jiang** [comment]
@Azquelt I think I have addressed all of you comments. Please double check. Thanks

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc [hunk 5]
```diff
@@ -226,6 +226,65 @@ Implementations MAY support:
...
 Failure to do so, may result in a different `Tracer` and incorrect handling of the OpenTelemetry data.
 
+
+=== Supported OpenTelemetry API Classes
+
+Classes from the following API packages MUST be supported by implementations of this specification, though this specification does not prevent additional API classes from being supported. Implementations are allowed to pull in a more recent patch version of the API classes.
+
+### Tracing APIs
+
+- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-api/{otel-java-version}/io/opentelemetry/api/package-summary.html[io.opentelemetry.api](except `GlobalOpenTelemetry`)
```
Comment (Azquelt) [inline tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc:236]:
```suggestion
- https://www.javadoc.io/static/io.opentelemetry/opentelemetry-api/{otel-java-version}/io/opentelemetry/api/package-summary.html[io.opentelemetry.api] (except `GlobalOpenTelemetry`)
```

**Review by Azquelt — APPROVED** [review]

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc [hunk 6]
```diff
@@ -226,6 +226,65 @@ Implementations MAY support:
...
+
+The above packages have dependencies on the following packages which must be supported to the extent that they are required by the Autoconfigure SPI classes:
+
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/package-summary.html[io.opentelemetry.sdk.trace]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/data/package-summary.html[io.opentelemetry.sdk.trace.data]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/export/package-summary.html[io.opentelemetry.sdk.trace.export]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/samplers/package-summary.html[io.opentelemetry.sdk.trace.samplers]
+  - https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-common/{otel-java-version}/io/opentelemetry/sdk/common/package-summary.html[io.opentelemetry.sdk.common]
+
+==== `WithSpan` annotation
```
Comment (Azquelt) [inline tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc:274]: Had one thought, should this heading be "Annotations" to fit better with the other headings?

**Review by Azquelt — APPROVED** [review]

File: tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc (hunk 6 above)
Comment (Azquelt) [reply tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc]: Sorry, I wasn't clear, I meant the whole heading should be

```
==== Annotations
```

**Review by Azquelt — APPROVED** [review]

**Pull request merged**